src/common-utils/main.sh

```diff
@@ -543,8 +542,11 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
 
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
-            echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
-            sed -i -e 's/ZSH_THEME=.*/ZSH_THEME="devcontainers"/g' ${user_rc_file}
+            if [ ! -f "$user_rc_file" ]; then
+                echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
```

Comment:
Now if the user has `user_rc_file` file, then this will not add the zshrc template. This doesn't make sense! 

Looking at https://github.com/devcontainers/features/issues/1035, the user can control and avoid this overwriting by setting `installOhMyZshConfig ` to `false`.

Instead of appending, let's prepend the contents to the beginning of the file. This ensures, the file will still prioritize user contents, but will have the necessary contents from this repo.

Reply:
what if user has this file but does not follow the template? in that case, still user has control over the content irrespective of appending the content in this file.

Reply:
I think that should be fine! User contents take priority over the content the Feature adds in

Reply:
as suggested, kindly review the changes done for this.